client/src/containers/App/App.js

```diff
@@ -8,9 +9,10 @@ import SilverStripeComponent from 'lib/SilverStripeComponent';
 class App extends SilverStripeComponent {
   render() {
     // TODO re-add <div className="app"> wrapper when applying to document.body
-    const Child = React.Children.only(this.props.children);
-    return (Child);
+    return <div>{this.props.children}</div>;
```

Comment:
Yeah, I haven't seen any issues yet, but I have noted it as a temporary hack that needs to be reviewed before merge.